src/seedu/addressbook/AddressBook.java

```diff
@@ -530,11 +530,7 @@ private static boolean isDeletePersonArgsValid(String rawArgs)
 	{
 		try
 		{
```

Comment:
flower bracket not formatted according to coding standards. Please follow coding standards closely.